lib/engine-addon.js

```diff
@@ -403,33 +400,21 @@ module.exports = {
           external: ['ember-engines/routes']
         });
 
-        // Babelify, but only to the extent of converting modules.
-        var babelOptions = {
```

Comment:
Keeping this around as a fallback should make it compat with non-master ember-cli.

I'm thinking something like:

    var babelOptions = (this.options && this.options.babel) || { /* current pojo here */}
      
    }